Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -53,6 +85,30 @@
     ],
     "defaultName": "HttpTriggerCSharp",
     "postActions": [
+        {
+            "condition": "NetCore",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
+            "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+            "ContinueOnError": "true",
+            "ManualInstructions": [],
+            "args": {
+                "referenceType": "package",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "1.1.0",
+                "projectFileExtensions": ".csproj"
+            }
+        },
+        {
+            "condition": "NetCore",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers Nuget package",
```

Comment:
We shouldn't need an explicit reference here. The extension package should bring this along.